behat.yml.dist

```diff
@@ -13,16 +13,13 @@ default:
             elements:
               about dropdown menu: "#nav-menu-expandable-group-2 .ecl-navigation-menu__links"
               breadcrumb: "nav.ecl-breadcrumb"
-              corporate bottom block: ".demo-block-footer-corporate-bottom"
-              corporate top block: ".demo-block-footer-corporate-top"
-              custom block: ".demo-block-footer-custom"
+              corporate footer: "footer.ecl-footer div.ecl-footer__corporate"
               language switcher: ".ecl-language-list"
               language switcher link: ".ecl-lang-select-sites__link .ecl-lang-select-sites__label"
               language switcher overlay: ".ecl-dialog__body"
               logo: "a.ecl-logo"
               page header: ".ecl-page-header__body"
               priorities dropdown menu: "#nav-menu-expandable-group-3 .ecl-navigation-menu__links"
-              search box: ".demo-block-header-search"
```

Comment:
This seems to be out of scope, why are we removing it here?

Reply:
It looks like this PR missed attention: https://github.com/openeuropa/oe_theme/pull/176
So I did the same within this PR.